run_brer/run_config.py

```diff
@@ -377,8 +377,12 @@ def __train(self, tpr_file=None, **kwargs):
             # of alpha.
             # noinspection PyUnresolvedReferences
             current_alpha = context.potentials[i].alpha
+            if current_alpha == 0.0:
+                raise Exception('Alpha value was constrained to 0.0')
```

Comment:
It would be okay to raise a `RuntimeError`. The base `Exception` is too low-level. You really shouldn't raise it directly.

Reply:
```suggestion
                raise ValueError('Read a non-sensical alpha value: 0.0')
```

Reply:
you prefer "constrained" to "trained"?